app/services/fixity_check_failure_service.rb

```diff
@@ -0,0 +1,75 @@
+class FixityCheckFailureService
+  include ActionView::Helpers::UrlHelper
+
+  attr_reader :fixity_check
+
+  def initialize(fixity_check)
+    raise ArgumentError.new("Fixity check passed was nil.") if fixity_check.nil?
+    raise ArgumentError.new("Please pass in a FixityCheck.") unless fixity_check.is_a? FixityCheck
+    @fixity_check = fixity_check
+    @asset = fixity_check.asset
+  end
+
+  def send
+    ActionMailer::Base.mail(from: "no-reply@sciencehistory.org",
+                            to: "digital-tech@sciencehistory.org",
```

Comment:
Ideally email addressees woudln't be hard-coded in, but would be from local_env.yml somewhere. Not sure if we already have keys for these though, not that big a deal to hard-code them in.

Reply:
Fixed!